persistence/src/commonMain/db_user/com/wire/kalium/persistence/Conversations.sq

```diff
@@ -375,6 +375,16 @@ selectConversationDetailsWithEventsFromSearch:
 SELECT * FROM ConversationDetailsWithEvents
 WHERE
     archived = :fromArchive
+    AND CASE
+        -- When filter is NONE, do not apply additional filters on conversation type
+        WHEN :conversationFilter = 'NONE' THEN 1 = 1
+        -- When filter is GROUPS, filter only group conversations
+        WHEN :conversationFilter = 'GROUPS' THEN type = 'GROUP'
+        -- When filter is ONE_ON_ONE, filter only one-on-one conversations
+        WHEN :conversationFilter = 'ONE_ON_ONE' THEN type = 'ONE_ON_ONE'
+        -- When filter is FAVORITES (future implementation)
+        ELSE 1 = 1
```

Comment:
suggestion: not a blocker but maybe we should return 0 for FAVORITES for now since there are no favourites yet so it would be more accurate with current situation 🤔